database/seeds/SectorFileIssuesTableSeeder.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use App\Models\User\Admin;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+
+class SectorFileIssuesTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
```

Comment:
Should this have an empty function body?

Reply:
Yeah - at the moment it's just there as a thing to purge the tables during a seed operation :)